### PR TITLE
[docs] Added docs and templates

### PR DESCRIPTION
**Pull request by EmperorYP7**
New documentation for project ethics and PR/Issue templates added.
Kindly review it once.

### CODE REVIEW

**Comment by EmperorYP7** [comment]
Some minor typos were intentionally left unchecked for good first issues.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+### Solves Issue #<number>
```
Comment (ankiiitraj) [inline .github/PULL_REQUEST_TEMPLATE.md:1]: This line is to be mentioned in description itself and not in title <--- Mention this

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+Description: <provide a description>
+
+#### Checklist
+- [ ] I've followed the Contributing guidelines provided in the repository.
+- [ ] I've made the changes which were demanded in the linked issue.
+- [ ] I've tested my code on a Chromium based browser.
```
Comment (ankiiitraj) [inline .github/PULL_REQUEST_TEMPLATE.md:8]: Mention Firefox too

File: .github/ISSUE_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
+## Issue #<number>
```
Comment (ankiiitraj) [inline .github/ISSUE_TEMPLATE.md:1]: issue number is assigned itself, no need to mention it here

**Review by ankiiitraj — CHANGES_REQUESTED** [review]
Just small changes and you are good to go

**Comment by ankiiitraj** [comment]
@prskid1000 Seems good to me after the changes.
Review this and we can merge after that

File: .github/ISSUE_TEMPLATE.md (hunk 3 above)
Comment (EmperorYP7) [reply .github/ISSUE_TEMPLATE.md:1]: Copy paste really got me here! 🤣

**Review by prskid1000 — CHANGES_REQUESTED** [review]
Check for grammatical error in Contributing Guide
It will be-

If you have worked on......

**Comment by EmperorYP7** [comment]
> Check for grammatical error in Contributing Guide
> It will be-
> 
> If you have worked on......

I thought, we should leave it for good first issues. We'll also get a clear picture if people are reading the docs carefully.

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (EmperorYP7) [reply .github/PULL_REQUEST_TEMPLATE.md:1]: Just to make sure we're on the same page.
```
### Solves Issue #<number>

Description: <provide a description> 
Title should be small and relevant.
```

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 2 above)
Comment (EmperorYP7) [reply .github/PULL_REQUEST_TEMPLATE.md:8]: Done.

File: .github/ISSUE_TEMPLATE.md (hunk 3 above)
Comment (EmperorYP7) [reply .github/ISSUE_TEMPLATE.md:1]: Done.

**Review by ankiiitraj — COMMENTED** [review]
lgtm

**Pull request merged**